runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -2236,20 +2234,6 @@ static void verifyDoFnSupportedStreaming(DoFn<?, ?> fn) {
   }
 
   static void verifyDoFnSupported(DoFn<?, ?> fn, boolean streaming) {
-    if (DoFnSignatures.usesSetState(fn)) {
```

Comment:
Thanks for the support status confirmation! I would image we still want to keep this checking part for runner v2 as well.

Reply:
We should still throw exceptions when using runner v2. Could you please add this check back for runner_v2? @reuvenlax

Reply:
done

Reply:
Thanks!